Add information on killing MeiliSearch (#690)
meilisearch/documentation: faq/README.md

- [I am trying to add my documents but I keep receiving a `400 - Invalid data` response.](/faq/README.md#i-am-trying-to-add-my-documents-but-i-keep-receiving-a-400-invalid-data-response)
- [My document upload failed with the `document id is missing` error.](/faq/README.md#my-document-upload-failed-with-the-document-id-is-missing-error)
- [I have uploaded my documents, but I get no result when I search in my index.](/faq/README.md#i-have-uploaded-my-documents-but-i-get-no-result-when-i-search-in-my-index)

> - [Is killing Meilisearch process safe?](/faq/README.md#is-killing-meilisearch-process-safe)

Reviewer: ```suggestion
- [Is killing a MeiliSearch process safe?](/faq/README.md#is-killing-a-meilisearch-process-safe)
```

Your documents upload probably failed.
To understand what happened, please check this [answer](/faq/README.md#i-did-a-call-to-an-api-route-and-i-only-got-an-updateid-as-a-response-what-does-it-mean).

## Is Killing MeiliSearch process safe?

> Killing MeiliSearch even in the middle of a process (ex: adding a batch of document) is **safe**. When you restart the server, it will start the task all over again.

Reviewer: ```suggestion
Killing MeiliSearch is **safe**, even in the middle of a process (ex: adding a batch of documents). When you restart the server, it will start the task from the beginning.
```
